### ISSUE TITLE
[Bug]: Unable to add EnOcean Device

### ISSUE THREAD

**Issue by HarterHorst** [Jey-Cee/ioBroker.enocean#74]
### What happened?

I'd like to add a device and it does not work. 

1. I press "Add device" in the Instance Settings dialog to add a device
2. I configure my device Manufacturer "Nodeon" and device (motion sensor 2-1-01) and press "add device"
3. A screen appears and tells me to 1. "click on add device" and 2 "start teach-in on device". 
4. When i click on "add device" the dialog disappears and i'm back in the Instance Settings dialog.
5. when I stark the teach-in on the device noting happens. 
 
EnOcean DOSMUNG USB Stick with SMA port configured on /dev/ttyusb0. Base ID has been recognized correctly. See log below. Any settings to set logging to next verbose level, because message ["/dev/ttyUSB0"] is leading nowhere.

### Adapter Version

0.5.1

### JS-Controller Version

3.3.15

### Node JS Version

v12.22.4

### NPM Version

6.14.14

### Operating System

Debian

### Relevant log output

```shell
enocean.0
2021-08-08 22:25:44.649	info	state enocean.0.gateway.teachin deleted
enocean.0
2021-08-08 22:25:44.642	info	Teachin mode de-activated
enocean.0
2021-08-08 22:25:24.472	info	["/dev/ttyUSB0"]
enocean.0
2021-08-08 22:24:32.261	info	["/dev/ttyUSB0"]

- Restart of adapter -
- 


enocean.0 | 2021-08-08 22:42:23.034 | info | {"BaseID":"ffdec800","Frequency":"868.3 Mhz","Protocol":"ERP1","AppVersion":"2.15.0.0","ApiVersion":"2.6.9.0","ChipID":"058cbd90","ChipVersion":"454f0103","AppDescription":"GATEWAYCTRL"}
-- | -- | -- | --
enocean.0 | 2021-08-08 22:42:22.476 | info | starting. Version 0.5.1 in /opt/iobroker/node_modules/iobroker.enocean, node: v12.22.4, js-controller: 3.3.15
host.inteltest1 | 2021-08-08 22:42:20.062 | info | instance system.adapter.enocean.0 started with pid 1091



```

**Comment by Jey-Cee** [comment]
There is an error that prevents teachin. I am on it.

**Comment by Jey-Cee** [comment]
Please check version 0.5.3

**Comment by Jey-Cee** [comment]
Please try version 0.5.3

**Comment by HarterHorst** [comment]
Teachin works. Thanks.

**Comment by HarterHorst** [comment]
Hi, hate to say it but the same bug resurrected from the dead.

Node.js - v12.22.8
iJS-Controller Version - 3.3.22
EnOcean Adapter: 0.5.4

I reinstalled everything to make sure no legacy configuration exists, but I cannot teach-in. I tried version 0.6.0 as well. After I have selected the manufacturer (nodon) and the device (micro smart plug) I press "Add Device" and the window just closes. 

btw: The sponsors are not readable in the dark mode and you forgot to mention my name on the list. You have received a little donation via paypal on August 8th, 2021. 8-)
